docs/content/about/configuration.md

```diff
@@ -615,7 +622,37 @@ security.
 | `rootcertbundle` | yes | The absolute path to the root certificate bundle. This bundle contains the public part of the certificates used to sign authentication tokens. |
 | `autoredirect`   | no      | When set to `true`, `realm` will automatically be set using the Host header of the request as the domain and a path of `/auth/token/`(or specified by `autoredirectpath`), the `realm` URL Scheme will use `X-Forwarded-Proto` header if set, otherwise it will be set to `https`. |
 | `autoredirectpath`   | no      | The path to redirect to if `autoredirect` is set to `true`, default: `/auth/token/`. |
-
+| `signingalgorithms`  | no      | A list of token signing algorithms to use for verifying token signatures. If left empty the default list of signing algorithms is used. Please see below for allowed values and default. |
+
+Available `signingalgorithms`:
+- EdDSA
+- HS256
+- HS384
+- HS512
+- RS256
+- RS384
+- RS512
+- ES256
+- ES384
+- ES512
+- PS256
+- PS384
+- PS512
+
+Default `signingalgorithms`:
```

Comment:
NOTE: these are the same as the available ones; I decided to keep them as is because god knows what token servers our users are doing; from what I've seen the end users don't understand them that well -- we should strive to make this as b/w compact as possible and iterate based on user feedback

Reply:
Isn't 3.0.0 a chance to break compat?

Is there a default list from go-jose that has been more vetted than ours?

Reply:
> Is there a default list from go-jose that has been more vetted than ours?

Actually there isn't no, I did look for it indeed.